lib/pages/search/search.dart

```diff
@@ -225,15 +225,19 @@ class SearchPage extends HookConsumerWidget {
                   ],
                 ),
                 Expanded(
-                  child: AnimatedSwitcher(
-                    duration: const Duration(milliseconds: 300),
-                    child: switch (selectedChip.value) {
-                      "tracks" => const SearchPageTracksTab(),
-                      "albums" => const SearchPageAlbumsTab(),
-                      "artists" => const SearchPageArtistsTab(),
-                      "playlists" => const SearchPagePlaylistsTab(),
-                      _ => const SearchPageAllTab(),
-                    },
+                  child: GestureDetector(
+                    onTap: () => FocusScope.of(context).unfocus(),
+                    behavior: HitTestBehavior.translucent,
```

Comment:
I don't think translucent is what we want here (I think the default is correct). See the options here:

https://api.flutter.dev/flutter/rendering/HitTestBehavior.html